pkg/asset/installconfig/kubevirt/client.go

```diff
@@ -202,6 +203,15 @@ func (c *client) GetNetworkAttachmentDefinition(ctx context.Context, name string
 	return c.getResource(ctx, namespace, name, nadRes)
 }
 
+func (c *client) GetHyperConverged(ctx context.Context, name string, namespace string) (*unstructured.Unstructured, error) {
```

Comment:
This call need aditional permission "Get hyperconvergeds"
Is this madatory?
If yes, need to add relevant permission to the role example: https://github.com/openshift/installer/blob/8fdf359d8edbc0ad52a0fe3adaf801c56f28327a/docs/user/kubevirt/role.yaml#L2
If not, the validation shouldn't fail in case of "Not Authorized error"

Reply:
what do u mean by additional permission? by whom? What role? The team told me that the credentials we use are the credentials used by the user that executes the installer.

Reply:
@nirarg as we discussed, the permissions validation and example will be handled in separate task.
I opened [Validate user permissions during tenant install](https://issues.redhat.com/browse/CNV-10142)